circus-train-s3-s3-copier/src/main/java/com/hotels/bdp/circustrain/s3s3copier/S3S3Copier.java

```diff
@@ -173,8 +172,16 @@ private void initialiseAllCopyRequests() {
         initialiseCopyJobs(subLocation, targetS3Uri);
       }
     }
-    LOG
-        .info("Finished initialising {} copy job(s)", copyJobRequests.size());
+
+    int totalCopyJobs = copyJobRequests.size();
```

Comment:
I use the totalCopyJob request number to make a determination about what the thread pool size should be in the `determineThreadPoolSize` method. I figured since we use it twice it could be worth throwing into a variable.

Reply:
Ah, OK, I didn't spot that, sorry, you're right.

Reply:
:+1: